test/ERC20VotesPartialDelegationUpgradeable.t.sol

```diff
@@ -171,6 +323,177 @@ contract Delegate is PartialDelegationTest {
     assertCorrectVotes(delegations, _amount);
   }
 
+  function testFuzz_EmitsDelegateChangedEvent(address _actor, uint256 _amount, uint256 _n, uint256 _seed) public {
+    vm.assume(_actor != address(0));
+    _amount = bound(_amount, 0, type(uint208).max);
+    _n = bound(_n, 1, tokenProxy.MAX_PARTIAL_DELEGATIONS());
+    PartialDelegation[] memory delegations = _createValidPartialDelegation(_n, _seed);
+    vm.startPrank(_actor);
+    tokenProxy.mint(_amount);
+
+    for (uint256 i; i < delegations.length; i++) {
+      vm.expectEmit();
+      emit DelegateChanged(_actor, delegations[i]._delegatee, delegations[i]._numerator);
+    }
+
+    tokenProxy.delegate(delegations);
+    vm.stopPrank();
+  }
+
+  function testFuzz_EmitsDelegateVotesChanged(address _actor, uint256 _amount, uint256 _n, uint256 _seed) public {
+    vm.assume(_actor != address(0));
+    _amount = bound(_amount, 0, type(uint208).max);
+    _n = bound(_n, 1, tokenProxy.MAX_PARTIAL_DELEGATIONS());
+    PartialDelegation[] memory delegations = _createValidPartialDelegation(_n, _seed);
+    vm.startPrank(_actor);
+    tokenProxy.mint(_amount);
+
+    DelegationAdjustment[] memory _votes = tokenProxy.exposed_calculateWeightDistribution(delegations, _amount);
+
+    for (uint256 i; i < delegations.length; i++) {
```

Comment:
replace with the following?

```
 _expectEmitDelegateVotesChangedEvents(_amount, new PartialDelegations[](0), delegations)
```

Reply:
Repleaced it here: https://github.com/voteagora/ERC20VotesPartialDelegationUpgradeable/pull/43/commits/4d24a3e168b6cf9a8adc751b831368457e4fbb37